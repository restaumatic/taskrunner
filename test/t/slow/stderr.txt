echo "This will appear on stdout"
echo "This will appear on stderr" >&2
# Crude attempt at sequencing
sleep 0.1
echo "stdout again"
echo "stderr again" >&2
