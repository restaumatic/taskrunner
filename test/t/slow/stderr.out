-- output:
[toplevel] stdout | This will appear on stdout
[toplevel] stderr | This will appear on stderr
[toplevel] stdout | stdout again
[toplevel] stderr | stderr again
